src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-cart"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Product img"
+        class="img"
+      />
+      <h2 class="product-cart__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
```

Comment:
The class name 'product-cart__name' should be 'product-card__name' to maintain consistency with the BEM methodology.